core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/Qualification.scala

```diff
@@ -149,8 +149,12 @@ class Qualification(outputPath: String, numRows: Int, hadoopConf: Configuration,
       val appResult = QualificationAppInfo.createApp(path, hadoopConf, pluginTypeChecker,
         reportSqlLevel, mlOpsEnabled, penalizeTransitions)
       val qualAppResult = appResult match {
-        case Left(errorMessage: String) =>
-          // Case when an error occurred during QualificationAppInfo creation
+        case Left(FailureApp("skipped", errorMessage)) =>
+          // Case when encountered DataBricks Photon event log during QualificationAppInfo creation
```

Comment:
nit: This block looks generic (could be skipping due to any reason). Can we update the comment as:
```suggestion
          // Handles cases to be skipped, e.g., encountering DataBricks Photon event logs
```

Reply:
Thanks, updated this!